### DISCUSSION TITLE
Client whitelist and max connection server fields not getting updated after editing a config file

### DISCUSSION THREAD

**Discussion by RahulBorana94** [ThreeMammals/Ocelot#2139] [General]
Hello,
we are currently facing an issue while configuring the `ClientWhitelist` setting in `RateLimitOptions` and in the `MaxConnectionsPerServer` setting in `HttpHandlerOptions` configuring is not getting updated at the time of merging of the configuration file 
the issue we are facing is mentioned below
 
### Client Whitelist for Rate Limit Options
```json
  "RateLimitOptions": {
        "ClientWhitelist": [ "test1", "test2" ],
        "EnableRateLimiting": true,
        "Period": "60s",
        "PeriodTimespan": 60,
        "Limit": 30
  },
 ```

### Max Connection Server For Http Handler Options 
```json
  "HttpHandlerOptions": {
        "AllowAutoRedirect": true,
        "MaxConnectionsPerServer": 500,
        "UseCookieContainer": true,
        "UseTracing": true
  },
```
We are working on merging the partial configuration file. On creation of the default `ocelot.json` file

**Comment by raman-m** [comment]
Hello Rahul,
Please upload the entire solution to your GitHub repositories to allow us to review it. We need to examine the complete solution rather than copied and pasted parts of your gateway application. We need to clone your repository and open the solution in Visual Studio. Following that, we will provide you with recommendations or accept the issue. Thus, we await your upload.
